Assets/Scripts/UI/PickupPup/DogProfile/DogProfile.cs

```diff
@@ -36,6 +36,8 @@ public DogProfileButtonController buttonController
     Image collarImage;
 	[SerializeField]
     UIElement[] descriptionFields; // Normal description must come first (not special)
+    [SerializeField]
+    AffectionMeter affectionMeter;
```

Comment:
Should be added to the subclass: `AdoptedDogProfile` because dogs in Shelter don't have an affection level